free-courses-en.md

```diff
@@ -303,7 +303,7 @@
 * [Java Online Training | Edureka](https://www.youtube.com/watch?v=hBh_CC5y8-s) - (Edureka)
 * [Java Programming: Solving Problems with Software](https://www.coursera.org/learn/java-programming) - (Coursera)
 * [Learn how to program: Java](https://www.learnhowtoprogram.com/java-june-2017) - Epicodus Inc.
-* [My Free Java Course - Automation, Development](https://angiejones.tech/free-java-course-test-automation/) - Angie Jones
+* [My Free Java Course - Automation, Development](https://angiejones.tech/free-java-course-test-automation/)  - Angie Jones (Applitools)
```

Comment:
The link you placed here looks more like a blog post pointing to the course material, not to the course itself.
Could we use the link that takes the user directly to the course instead? 

Also, please use the name as given from the source.

```suggestion
* [Java Programming](https://testautomationu.applitools.com/java-programming-course/) - Angie Jones (Applitools)
```

Reply:
Absolutely!

Reply:
Thank you for the fix!
Just a small thing as well, sorry I didn't say it explicitly, I only put it in the suggestion.

Could you remove the extra space around the hyphen? (Before the author's name.)

Reply:
No worries, I am still learning how to use github. I appreciate your feedback.